cockroachcloud/egress-perimeter-controls.md

```diff
@@ -64,18 +62,19 @@ The operations described in this page require an API key with very broad permiss
     {% include_cached copy-clipboard.html %}
     ~~~shell
     curl --request GET \
-      --url "https://management-staging.crdb.io/api/v1/clusters/$CLUSTER_ID" \
-      --header "Authorization: Bearer $CC_API_KEY"
+    --header "Authorization: Bearer $CC_API_KEY" \
+    --url "https://management-staging.crdb.io/api/v1/clusters/$CLUSTER_ID"
     ~~~
 
     ~~~json
+    
     {
-      "id": "f0d06734-55ba-42e2-b606-2c1f58e6b9eb",
-      "name": "able-quetzal",
-      "cockroach_version": "v22.1.8",
+      "id": "21f474d5-3e65-4a54-9317-e8d8803ef917",
+      "name": "docstest",
+      "cockroach_version": "latest-v22.2-build(sha256:e42c4de8577556132120a9ab07efc1a2a96779c028ebab99223d862d9792428b)",
```

Comment:
Probably just a nit, but I wonder if this should be changed to the 22.1.9 build hash. Unless this feature requires v22.x.